proto/stakeibc/callbacks.proto

```diff
@@ -29,5 +29,5 @@ message ReinvestCallback {
 message UndelegateCallback {
   string hostZoneId = 1; 
   repeated SplitDelegation splitDelegations = 2;
-  repeated uint64 unbondingEpochNumbers = 3;
+  repeated uint64 epochUnbondingNumbers = 3;
```

Comment:
I agree, the original naming was a little clumsy, and `epochUnbondingNumber` is a little cleaner - but imo it also makes it a little less clear that this holds epoch numbers. Maybe there's a better name than either of those - wdyt?

Reply:
maybe: `epochUnbondingRecordNumbers`? I think it is clear that `epochUnbondingRecord`s are scoped to epochs, and the actual numbers are keys for that data type which is why I switched it. Wdyt?

Reply:
I think being in the weeds, it seems clear now, but I think being less fluent with the codebase, "number" being synonymous with key isn't obvious imo. Not a huge deal though. If you like any of these better, go ahead and update it:
* `epochUnbondingIds`
* `epochUnbondingRecordIds`
* `epochNumbers`

Otherwise feel free to keep it as `epochUnbondingNumbers` if you think that makes the most sense!

Reply:
I like `epochUnbondingRecordIds`!

Reply:
thanks for the suggestion